web-ui/src/components/member_selector/MemberSelector.jsx

```diff
@@ -176,9 +177,14 @@ const MemberSelector = ({
       >
         <CardHeader
           avatar={
-            <IconButton onClick={() => setExpanded(!expanded)}>
-              {expanded ? <ExpandLessIcon /> : <ExpandMoreIcon />}
-            </IconButton>
+            <ExpandMore
+              expand={expanded}
+              onClick={handleExpandClick}
+              aria-expanded={expanded}
+              aria-label="show more"
+            >
+              <ExpandMoreIcon />
```

Comment:
Why doesn't the `ExpandMore` component supply `ExpandMoreIcon` so it isn't needed in this file?